webpack.config.dev.js

```diff
@@ -0,0 +1,22 @@
+const merge = require('webpack-merge');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+let dev = {
```

Comment:
In fact the accepted answer on that question says exactly the same. Use `const` always unless you *really* need to change the value.